keras2onnx/ke2onnx/main.py

```diff
@@ -69,7 +69,7 @@ def convert_keras_flatten(scope, operator, container):
 
 def _apply_not_equal(oopb, target_opset, operator):
     if target_opset >= 11:
-        equal_out = oopb.add_node('Equal', [operator.inputs[0].full_name, np.array([0], dtype='float32')],
+        equal_out = oopb.add_node('Equal', [operator.inputs[0].full_name, np.array([operator.mask_value], dtype='float32')],
```

Comment:
In some case, mask_value will be None?

Reply:
Fair point. I raised an exception in this case, because it will only occur if the parsing does not work. I think it is preferable to have a clear error, than to silently use `0.0` as the default.